config.example/group_vars/k8s-cluster.yml

```diff
@@ -69,6 +69,11 @@ docker_insecure_registries: "{{ groups['kube-master']|map('regex_replace', '^(.*
 crio_insecure_registries: "{{ groups['kube-master']|map('regex_replace', '^(.*)$', '\\1:5000')|list + ['registry.local:31500']}}"
 docker_registry_mirrors: "{{ groups['kube-master'] | map('regex_replace', '^(.*)$', 'http://\\1:5000') | list }}"
 
+# TODO: Add support in containerd for automatically setting up registry
+# mirrors, not just the k8s-local registry
+containerd_insecure_registries:
+        "registry.local:31500": "http://registry.local:31500"
+
```

Comment:
Is there a reason this doesn't have the same format as the similar values above?

Reply:
The format for this variable was actually just changed in Kubespray! See release notes for v2.18.1: https://github.com/kubernetes-sigs/kubespray/releases/tag/v2.18.1

Interestingly, I originally misread this and added it too early, when I thought it applied to v2.18.0.

However, v2.18.1 appears to have bugfixes we need anyway to get insecure registries to work. I'm testing an update to that release now, in which case this change will still apply. :-)